### PR TITLE
Replacement of hot filehandlers for Mojo::Log

### PR DESCRIPTION
**Pull request by dasantiago**
This change will replace writing to the autoinst-log.txt with
a hot filehandler, with a proper logging mechanism: Mojo::Log

**This change will remove the color printing when printing to the terminal.**

### CODE REVIEW

File: bmwqemu.pm [hunk 1]
```diff
@@ -53,7 +54,7 @@ our $screenshotpath = "qemuscreenshot";
 
 # global vars
 
-our $logfd;
+our $logger;
 
 our $istty;
```
Comment (coolo) [inline bmwqemu.pm]: this variable is not obsolete

File: bmwqemu.pm [hunk 2]
```diff
@@ -182,55 +185,43 @@ sub get_timestamp {
...
 
-sub print_possibly_colored {
-    my ($text, $color) = @_;
-
-    if (($direct_output && !$istty) || !$direct_output) {
-        $logfd && print $logfd get_timestamp() . "$$ $text\n";
-    }
-    if ($istty || !$logfd) {
-        if ($color) {
-            print STDERR colored(get_timestamp() . "$$ " . $text, $color) . "\n";
```
Comment (coolo) [inline bmwqemu.pm]: as you remove colored calls, you can remove the Term::ANSIColor dependency as well

File: bmwqemu.pm (hunk 1 above)
Comment (dasantiago) [reply bmwqemu.pm]: Good catch!

Change done :-)

File: bmwqemu.pm (hunk 2 above)
Comment (dasantiago) [reply bmwqemu.pm]: Done.

Removed from the imports and from the cpanfile

**Comment by okurz** [comment]
Hi, I think with this PR you changed the timestamp format missing out on the subsecond precision. Was this changed on purpose or maybe missed?

File: bmwqemu.pm (hunk 2 above)
Comment (dasantiago) [reply bmwqemu.pm]: @okurz Only for STDOUT/STDERR, correct?

If yes, I'm fine with it. Please open a poo.

Comment (okurz) [reply bmwqemu.pm]: I did with https://progress.opensuse.org/issues/34978 , thx

**Pull request merged**